docs/content/get-started/open-project.md

```diff
@@ -2,6 +2,10 @@
 
 Launch the Unity Editor. It should automatically detect the project but if it doesn't, select **Open** and then select `gdk-for-unity-fps-starter-project/workers/unity`.
 
+<%(Callout type="warning" message="
+It will take about 10 minutes for Unity to load the Starter Project for the first time. In the meantime, 
+why not take a stroll on our [Games Blog](https://improbable.io/games/blog)?" )%>
```

Comment:
There's too many colourful messages and so very many warning messages (whatever is the latest thing we notice), so better to keep it as a `>`.
Also, no need for predictions on the future with "it will take". Keep it present simple as a universal fact ("it takes"). It makes the text stronger.
We say "open" a Unity project in the Unity Editor, rather than "load" a project.
```suggestion
>The first time you open the Starter Project in your Unity Editor, It takes about 10 minutes; it's much quicker to open after this. (While you are waiting, you could look at our [Games Blog](https://improbable.io/games/blog).)
```

Reply:
Note to self - added to new PR